src/loki/status_action.cc

```diff
@@ -1,8 +1,46 @@
+#include <boost/property_tree/json_parser.hpp>
+#include <iostream>
+#include <sstream>
+
+#include "baldr/tilehierarchy.h"
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+
+  auto* status = request.mutable_status();
+
+  std::stringstream ss;
+  boost::property_tree::json_parser::write_json(ss, config);
+  status->set_config(ss.str());
+
+  // only return more info if explicitly asked for (can be very expensive)
+  if (request.options().verbose()) {
+    // get _some_ tile
+    static baldr::graph_tile_ptr tile = nullptr;
+    if (!tile) {
+      for (const auto& tile_id : reader->GetTileSet()) {
+        tile = reader->GetGraphTile(tile_id);
+        if (tile->id().level() < baldr::TileHierarchy::GetTransitLevel().level &&
+            tile->header()->nodecount() > 0) {
+          break;
+        }
+      }
+    }
+
+    static std::string bbox = "";
+    if (bbox == "" && connectivity_map) {
+      bbox = connectivity_map->to_geojson(2);
```

Comment:
One day I want to make the map use the spatial index since it has even higher resolution, maybe not interest for this endpoint yet but showing an optional 'coverage' geojson might be pretty cool in the future